simulationfactory/src/pages/Loginsignuppage.js

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import CreateStyles from "../util/stylesheet";
+import Grid from "@material-ui/core/Grid";
+import Card from "@material-ui/core/Card";
+import CardActions from "@material-ui/core/CardActions";
+import CardContent from "@material-ui/core/CardContent";
+import { List, ListItem, ListItemText, Typography } from "@material-ui/core";
+import TextField from '@material-ui/core/TextField';
+
+import Topbar from "../components/Topbar";
+import Navbar from "../components/Navbar";
+
+import {RegisterRoutes} from "../util/RouteBuilder";
+
+
+function Loginsignuppage() {
+
+    const Styles = CreateStyles();
+
+    return (
+        <div className={Styles.root}>
+            <Topbar message="Login/Signup!"/>
+            <Navbar/> {/* This is necessary for some styling reason I'm too backend to understand */}
+            <main className={Styles.content}>
+                <div className={Styles.toolbar} />
+                <Grid container spacing={3}>
+                    <Grid item xs={12} sm={6}>
+                        <Card className={Styles.root}>
+                            <CardContent>
+                                <Typography className={Styles.title} variant="h2">
+                                    Login
+                                </Typography>
+                                <div ClassName={Styles.list}>
+                                    <List>
+                                        <ListItem>
+                                            <ListItemText primary="User ID: " />
+                                                <form className={Styles.root} noValidate autoComplete="off">
+                                                    <TextField id="outlined-login-user-id" label="User ID" variant="outlined" />
+                                                </form>
+                                        </ListItem>
+                                        <ListItem>
+                                            <ListItemText primary="Password: " />
+                                                <form className={Styles.root} noValidate autoComplete="off">
+                                                    <TextField id="outlined-login-user-password" label="Password" variant="outlined" />
+                                                </form>
+                                        </ListItem>
```

Comment:
Should probably have these dot out, so you can't see the text.
But this is a mockup, so I'm still going to merge. We'll just have to keep an eye on that when we go over this again.